nossas/slate-editor
# Feature handle image plugin with modal

# Related issues
- Generating duplicate IDs slate-image-plugin-button #6
- Image has been duplicated when add link #16
- Create a feature to open a modal when insert a link #17

# How to test
- Insert an image
- Hover it, a layer to edit the image data should be shown
- Click on the layer, a modal should be shown
- Fill the form fields and test the action buttons
